device-motion.html

```diff
@@ -0,0 +1,95 @@
+<!--
+@license
+Copyright (c) 2017 Abdón Rodríguez Davila (@abdonrd). All rights reserved.
+This code may only be used under the MIT style license found at https://abdonrd.github.io/LICENSE.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+Polymer element to detecting device motion.
+
+Example to get the device motion values:
+
+```html
+<device-motion motion="{{motion}}"></device-motion>
+```
+
+@demo demo/index.html
+-->
+
+<dom-module id="device-motion">
+  <script>
+    Polymer({
+      is: 'device-motion',
+
+      properties: {
+        /**
+         * The device motion data.
+         *
+         * An object with the `accelerationX`, `accelerationY`, `accelerationZ`,
+         * `accelerationIncludingGravityX`, `accelerationIncludingGravityY`, `accelerationIncludingGravityZ`,
+         * `rotationRateAlpha`, `rotationRateBeta`, `rotationRateGamma` and `interval` values.
+         */
+        motion: {
+          type: Object,
+          readOnly: true,
+          notify: true
+        },
+
+        /**
+         * Indicates whether `DeviceMotionEvent` is supported or not.
+         */
+        apiSupported: {
+          type: Boolean,
+          readOnly: true,
+          notify: true
+        },
+
+        /**
+         * Indicates whether relevant sensors are available or not.
+         */
+        sensorsAvailable: {
+          type: Boolean,
+          readOnly: true,
+          notify: true
+        }
+      },
+
+      attached: function() {
+        if (window.DeviceMotionEvent) {
+          this._setApiSupported(true);
+          this._devicemotionListener = this._handleMotion.bind(this);
+          window.addEventListener('devicemotion', this._devicemotionListener, true);
+        } else {
+          this._setApiSupported(false);
+        }
+      },
+
+      detached: function() {
+        window.removeEventListener('devicemotion', this._devicemotionListener);
+      },
+
+      _handleMotion: function(event) {
+        this._setMotion({
```

Comment:
IMO, this is not a good idea. How do users find out that `acceleration` and `accelerationIncludingGravity` objects have `x`, `y` and `z` properties, but `rotationRate` object has `alpha`, `beta` and `gamma` properties, not `x`, `y` and `z`?

Frankly, I don't like both options. I believe this is should be individual properties. See #6.

What about merge this pull request "as is" and then implement #6?